web/client/components/app/withExtensions.js

```diff
@@ -164,6 +165,7 @@ function withExtensions(AppComponent) {
                                 reducersList.push(name);
                             });
                             store.storeManager.addEpics(impl.name, impl?.epics ?? {});
+                            store.storeManager.unmuteEpics(impl.name);
```

Comment:
just a question: do we need to unmute also if epics is empty?
I see that `useModulePlugins.js` is using

```js
if (size(impl.epics)) {
    store.storeManager.addEpics(impl.name, impl.epics);
    store.storeManager.unmuteEpics(impl.name);
}
```

is there a different behaviour here?

Reply:
No, it will behave identically. There is an extra check inside unmuteEpics to see if plugin has epics registrations, otherwise there is nothing to iterate. An overhead would be in an extra run of normalizeName(), nothing else.